lucille-examples/lucille-s3-ingest-example/conf/s3-opensearch.conf

```diff
@@ -0,0 +1,69 @@
+connectors: [
+ {
+    name: "fileConnector",
+    class: "com.kmwllc.lucille.connector.FileConnector",
+    pipeline: "pipeline1",
+    getFileACLs: true,
+    pathToStorage: ${PATH_TO_STORAGE}
+    cloudOptions: {
+      region: ${AWS_REGION}
+      accessKeyId: ${AWS_ACCESS_KEY_ID}
+      secretAccessKey: ${AWS_SECRET_ACCESS_KEY}
+    }
+ }
+]
+
+pipelines: [
+  {
+    name: "pipeline1",
+    stages: [
+     {
+       name:"TextExtractor"
+       class: "com.kmwllc.lucille.tika.stage.TextExtractor"
+       # FileConnector places the content in this "file_content" field.
+       byte_array_field: "file_content"
+       metadata_prefix: ""
+       metadata_blacklist: []
+       tika_config_path: "conf/tika-config.xml"       
+     },
+     {
+      name:"ChunkText"
+      class: "com.kmwllc.lucille.stage.ChunkText"
+      source: "text"
+      dest: "text"
+      chunking_method: "paragraph"
+     },
+     {
+      name: "EmitNestedChildren"
+      class : "com.kmwllc.lucille.stage.EmitNestedChildren"
+      drop_parent: "false"
+     },
+     {
+      name: "DeleteFields"
+      class : "com.kmwllc.lucille.stage.DeleteFields"
+      fields : ["file_content", "_version"]
+     }   
+    ]
+  }
+]
+
+
+indexer {
+  type: "OpenSearch"
+  batchTimeout: 1000
+  batchSize: 100
+  # uncomment below if you want message_id to replace id
+  # idOverrideField: "message_id"
+  sendEnabled: true
+}
+
+opensearch {
+  url: ${INGEST_URL}
+  index: ${INGEST_INDEX}
+  acceptInvalidCert: true
+}
+
+worker {
+  pipeline: "pipeline1"
```

Comment:
it's only necessary to provide worker.pipeline when starting a Worker in a separate JVM as part of a distributed mode deployment
When running lucille in "local" mode with all components in the same JVM, the Runner will create workers for each pipeline used in the run, and you don't need to specify worker.pipeline

Reply:
removed